recipes/skaware/files/HOST_CPU_x86_64/sysdeps.h

```diff
@@ -0,0 +1,49 @@
+/* ISC license. */
```

Comment:
are these files we get from upstream or do we have to figure out all these parameters ourselves to add a new arch? Would be nice with a small comment in the recipe about that